src/collar/oc_sys.lsl

```diff
@@ -500,6 +500,7 @@ RebuildMenu() {
     g_lAppsButtons = [] ;
     llMessageLinked(LINK_SET, MENUNAME_REQUEST, "Main", "");
     llMessageLinked(LINK_SET, MENUNAME_REQUEST, "Apps", "");
+    llMessageLinked(LINK_SET, MENUNAME_REQUEST, "AddOns", "");
```

Comment:
I verified that backward compatibility is present in the rest of the file. Thank you for fixing this.